### PR TITLE
tul/merge-into-clarin-dspace

### PR DESCRIPTION
**Pull request by milanmajchrak**
We want to update CLARIN-DSpace7.5. code according to TUL customization addons.

### CODE REVIEW

File: dspace/config/clarin-dspace.cfg [hunk 1]
```diff
@@ -174,3 +174,48 @@ statistics.cache-server.uri = http://cache-server.none
...
+# Add rdf here, if you are using dspace-rdf to export your repository content as RDF.
+# Add iiif here, if you are using dspace-iiif.
+event.dispatcher.default.consumers = versioning, discovery, eperson
+
+# Edit Item - Status option
+identifiers.item-status.register-doi = false
+
+##### Dataquest URL - sing in the footer #####
+themed.by.url = https://www.dataquest.sk/dspace
+themed.by.company.name = dataquest s.r.o.
```
Comment (MajoBerger) [inline dspace/config/clarin-dspace.cfg:221]: shouldnt here be a blank line

**Review by MajoBerger — CHANGES_REQUESTED** [review]
blank line at end of file

File: dspace/config/clarin-dspace.cfg (hunk 1 above)
Comment (milanmajchrak) [reply dspace/config/clarin-dspace.cfg]: Fixed, thank you

**Review by MajoBerger — APPROVED** [review]

**Review by milanmajchrak — COMMENTED** [review]
q

**Pull request merged**